tests/validation/tests/v3_api/test_cert_rotation.py

```diff
@@ -1,18 +1,28 @@
 import pytest
 import datetime
 import time
-from .common import get_user_client_and_cluster
-from .common import validate_cluster_state
-from .common import get_etcd_nodes
-from .common import wait_for_nodes_to_become_active
-from .common import wait_for_node_status
+import os
+from .common import *  # NOQA
```

Comment:
Why are we importing all instead of the required ones?

Reply:
This is because we end up needing many of the .common global variables like `TEST_RBAC = ast.literal_eval(os.environ.get('RANCHER_TEST_RBAC', "False"))` and `if_test_rbac = pytest.mark.skipif(TEST_RBAC is False,
                                  reason='rbac tests are skipped')`

Reply:
okay. But we could still import those rather than the whole file i believe. 
Its just best practice to import what is required rather than the whole file.

Reply:
Will do.

Reply:
Only imported what was required.